src/object/flattenObject.spec.ts

```diff
@@ -13,6 +13,8 @@ describe('flattenObject', function () {
       'a.b': 'yay'
     })
 
+    const date= new Date();
```

Comment:
Formatting: missing a space before the `=` sign.

Reply:
```suggestion
    const date = new Date();
```